Product Designer/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+       <title>
+            Product Designer
+       </title>
+       <link rel="stylesheet" href="/Product Designer/assets/css/styles.css" />
+    </head>
+        <body>
+        <!-- START : HEADER SECTION -->
+        <header class="mailSection">
+            <h6>shruthi.bhatt.siriuscom.com</h6>
+        </header>
+        <!-- END :  HEADER SECTION -->
+
+        <!-- START : NAME SECTION -->
+        <section class="nameSection">
+            <h1 class="name">
+                <span>Shruthi</span> Bhatt
```

Comment:
In this case `Bhatt` should have span to make the text bolder

Reply:
Made the change applied span to bhatt